local/languages/finna/en-gb.ini

```diff
@@ -771,37 +769,59 @@ Publications = "Publications"
+R2_menu_heading_register = "Access rights to Finna Reading Room"
+R2_monthly_limit_exceeded = "Limits have been set for monthly use for security reasons. If your session in Finna Reading Room is terminated before you finished what you were doing, contact us by sending a message to finna-palaute@helsinki.fi or continue using the service later. You still have access to the open resources. The amount of monthly use is stored in a tracking log. Unless unusual activity is detected, the logs are deleted after 30 days."
+R2_mylist_restricted = "Even if the metadata of certain materials were protected, you can add them to your Favourites list. However, protected data is not displayed to users who have not completed the user agreement form for the Reading Room."
+R2_register = "Log in to Finna Reading Room"
+R2_register_myresearch = "Fill in the user agreement form"
+R2_register_error = "Registration to Finna Reading Room failed."
+R2_register_form_datasource = "(information obtained from the Suomi.fi e-Identification service)"
+R2_register_form_title = "User agreement for content available in Finna Reading Room"
+R2_register_form_help_post_html = "Your information will be stored in the Finna user register and the licence service for REMS – Resource Entitlement Management System (CSC).<br>Read more in the <a href="%%url%%" data-lightbox-ignore>privacy statement</a>. Where necessary, information on the processing of personal data for use in scientific research can also be found on the Finlex.fi website."
+R2_register_form_help_pre_html = "<h4>Finna Reading Room terms of use for metadata</h4><p><strong>The metadata contained in Finna Reading Room (incl. descriptive metadata, catalogue metadata) may be used within the limits set by the GDPR and the Data Protection Act.</strong> The user has the right to browse and search for records. The user agrees to use the metadata and materials in accordance with the specified purpose.</p>"
+R2_register_form_help_returninguser_pre_html = "<p><a href="%%url%%" data-lightbox-ignore>Read more about Finna Reading Room</a></p>"
+R2_register_form_userinfo = "User information"
+R2_register_form_age = "I am over 15 years old"
+R2_register_form_license_accept = "I agree to the terms of use for metadata"
+R2_register_form_usage = "Your specified purpose for processing the metadata"
+R2_register_form_usage_help_html = "The user agrees to use the materials in accordance with the purpose specified when they logged in."
+R2_register_form_usage_help_tooltip_html = "<h4>Specified purpose</h4><p>Choose the option that best describes your purpose from the list below.</p><p>The law requires that you specify your purpose for processing the data every time you use protected metadata. The purpose must also be specified for metadata every time you use them.</p><p>The purpose must be specified regardless of the method of disclosure of data and whether or not the personal data register contains special category data.</p>"
```

Comment:
Lopun "whether or not the personal data register contains special category data.", avautuuko tämä käyttäjälle? Mä en ainakaan tajunnut, mihin "special category data" viittasi. Suomeksi on "sisältääkö henkilörekisteri erityisiä henkilötietoryhmiä koskevia henkilötietoja.", joka ei sekään ole selkeä, mutta osoittaa asian koskevan jotain henkilötietoja.

Reply:
special category data on suoraan lakitekstistä tulevaa kuten erityiset henkilötietoryhmätkin. Näiden lakijuttujen kanssa on hankala olla selkokielinen, koska merkitykset muuttuu saman tein

Reply:
Ok sitten.